### ISSUE TITLE
About student loss

### ISSUE THREAD

**Issue by daxiongpro** [Vegeta2020/SE-SSD#84]
Hello, thanks for your code. I find that in your loss:`loss = cls_loss_reduced + ious_loss + dir_loss + iou_pred_loss`, dir_loss and iou_pred_loss didn't introduced in your paper. `dir_loss ` only mentioned but does not give formular out. 
And I also find  many loss like "cls_pos_loss"  are same as your another repo "CIA-SSD", and I want to know is these loss necessary in SE-SSD, because you does not mentioned in SE-SSD

**Comment by Vegeta2020** [comment]
@daxiongpro All losses associated with hard targets are necessary to be kept to train the model, as they follow the design of SECOND and "dir_loss" is actually a part of location regresion loss. Besides, "iou_pred_loss" and its related consistency loss can be removed, but they can improve the performance of SE-SSD slightly.
